Answers.md

```diff
@@ -2,10 +2,25 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+    It is a javascript library among many others that try to make it easier for developers to build
+    large scale applications effectively and efficiently.
+
 1.  What does it mean to _think_ in react?
 
+    It means to see UI as a combination of smaller components and 
+    follow the data flow from the parent down to the smallest components.
+
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
 
+    Stateful components can have their own state, independent of its parent. It can handle state changes without having to 
+    get data from the parent. Functional components on the other hand rely on its parent to get data and add it to the DOM.
+

```

Comment:
Yes, class components also have access to life cycle methods. This is changing with hooks though.